holidayextras/brands
# Add more UIToolkit Mapping (weather, cards)

#### What does this PR do? (please provide any background)
We want to remove the UIToolkit less files from our projects. The LESS there was intended for standalone open source projects and the design of elements don't fit with our brands.
Previously we have mapped elements with bootstrap like-for-like with toolkit components. This PR add the styling for components which aren't present in bootstrap. Brand new components

#### What tests does this PR have?
None

#### How can this be tested?
I have another branch in the pattern library with the same name.
If you `npm link` pattern library to this brands repo, then navigate to `molecules` -> `Credit/Debit Cards`
You should see some nice credit / debit card logos.

AND/OR

You can boot up master of pattern library and view the planner page and see if the weather icons work correctly.

Any questions give me a shout

#### What domains are affected?
None

#### Screenshots / Screencast
<img width="304" alt="screen shot 2016-05-10 at 15 21 48" src="https://cloud.githubusercontent.com/assets/778942/15149820/ef74e6c6-16c2-11e6-9994-8a29a89e7fda.png">

#### What gif best describes how you feel about this work?
![Into the Map](http://cos.h-cdn.co/assets/15/29/1436979634-joey-in-map-o.gif)

---

**Notes when reviewing Static Site Generator projects, please think carefully about what can be done to centralise common functionality. Please also pay attention to anything configurable that will also need updating in the Yeoman generator template**


#### Reviewers

**Review 1**

**Review 2**

**Review 3**

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.
